apis/v1alpha1/defaulting.go

```diff
@@ -379,3 +375,48 @@ func getVersionFromImage(imageURL string) string {
 	}
 	return DefaultVersion
 }
+
+func defaultStartupProbe() *corev1.Probe {
+	// When StartupProbe is successful, the liveness probe and readiness probe will be enabled.
+	return &corev1.Probe{
+		ProbeHandler: corev1.ProbeHandler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path: DefaultHealthEndpoint,
+				Port: intstr.FromInt32(DefaultHTTPPort),
+			},
+		},
+		PeriodSeconds: 10,
```

Comment:
We can set PeriodSeconds to 5 and FailureThreshold to 60 to improve the startup speed.